src/Microsoft.Extensions.Configuration.AzureAppConfiguration/Extensions/TimeSpanExtensions.cs

```diff
@@ -8,7 +8,7 @@ namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.Extensions
 {
     internal static class TimeSpanExtensions
     {
-        private const int MaxAttempts = 63;
+        private const int MaxSafeShiftBit = 63;
```

Comment:
The new name is also a bit odd. 63 is not a "Bit" as the name suggests.
How about calling it `MaxShiftCount`?

Reply:
How about MaxShiftLimit?

Reply:
Discussed offline. We will use "SafeShiftLimit"